clients/search-component/src/utils/hooks/useChat.ts

```diff
@@ -44,6 +44,14 @@ export const useChat = () => {
       if (doneReading) {
         done = doneReading;
         setIsLoading(false);
+
+        // Get First Chat Container 
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const chatContainer: any = modalRef.current?.querySelector("div.chat-container");
+        if (chatContainer) {
+          chatContainer.style.height = "fit-content";
+        }
```

Comment:
Had to do a hack here to get the footer to be at the right position after a message is generated.